ExpressWash/ExpressWash/EditWasherViewController.swift

```diff
@@ -17,28 +17,33 @@ class EditWasherViewController: UIViewController {
     @IBOutlet weak var txtRateLarge: UITextField!
     @IBOutlet weak var txtRateMedium: UITextField!
     @IBOutlet weak var txtRateSmall: UITextField!
+    @IBOutlet weak var btnSave: UIButton!
 
     // MARK: - Properties
-    var washer: Washer? { didSet { updateViews() } }
+//    var washer: Washer? { didSet { updateViews() } }
```

Comment:
is this needed? if so, uncomment eventually

Reply:
It's not. Just missed deleting it. Thanks!